server/tests/api_v5/anonymous_account/test_account_send_email_validation.py

```diff
@@ -0,0 +1,37 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+
+from parsec._parsec import anonymous_account_cmds
+from tests.common import Backend, HttpCommonErrorsTester
+from tests.common.client import AccountRpcClient
+
+
+async def test_anonymous_account_account_send_email_validation_token_ok(
+    account: AccountRpcClient, backend: Backend, skip_if_postgresql: None
```

Comment:
I'd use `xfail_if_postgresql` which indicates this is temporary (see message in the fixture). We use `skip` for permanent skips.

```suggestion
    account: AccountRpcClient, backend: Backend, xfail_if_postgresql: None
```